CogNative/backend/translation/translation.py

```diff
@@ -0,0 +1,15 @@
+from googletrans import Translator
+
+
+class translation:
+
+    def __int__(self):
+        self.translator = Translator()
+
+    def translate(self, text, src, des):
+        result = self.translator.translate(text, src=src, dest=des)
+        return result.text
+
+    def translate_to(self, text, des):

```

Comment:
You might add a docstring to explain how this functions different. I'm assuming t just auto-detects src language?